src/main/java/users/controllers/UsersController.java

```diff
@@ -0,0 +1,11 @@
+package users.controllers;
+
+import static spark.Spark.get;
+
+public class UsersController {
+
+  public static void greetUser() {
+    get("/hello/:name ", (request, response) -> "Hello " + request.params(":name"));
```

Comment:
Cool! Now, we want to return an object called User, that contains two attributes, `name` and `lastName`.

Create a folder named `models` in the users folder. There, you create the class `User` and it should contain the attributes `name` and `lastName` and their respective getters. Hint: Take a look to the library `lombok` it can help you to define models easier.

Then, the endpoint /hello/ should receive name and last_name attibutes, instantiate an object of class `User` with those attributes, and response a JSON.

Example:
```
GET /hello?name=juan&last_name=perez
```

and should return
```
{
    name: juan,
    last_name: perez
}
```
Hint: Take a look to Gson to serialize objets to json and deserialize json to object.